### PR TITLE
Update to Airbase 80

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by findepi** [comment]
@electrum what does new airbase give wrt to Guava?

File: presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java [hunk 1]
```diff
@@ -215,7 +217,8 @@ else if (retryStatus.shouldStopRetry(e)) {
                 throw propagate(e);
             }
             log.warn(e, "Failed on executing %s with attempt %d, will perform async retry.", callableName, retryStatus.getAttempts());
-            return Futures.dereference(retryExecutorService.schedule(() -> runAsyncInternal(callableName, stats, callable, retryStatus), retryStatus.getRetryDelayInMs(), MILLISECONDS));
+            Callable<ListenableFuture<V>> retryCallable = () -> runAsyncInternal(callableName, stats, callable, retryStatus);
+            return transformAsync(retryExecutorService.schedule(retryCallable, retryStatus.getRetryDelayInMs(), MILLISECONDS), x -> x, directExecutor());
```
Comment (dain) [inline presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java]: `Function.identity()`

**Comment by electrum** [comment]
@findepi `24.0-jre` https://github.com/airlift/airbase/pull/140

File: presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java [hunk 2]
```diff
@@ -413,7 +415,8 @@ private QueryResult executeQuery(String url, String username, String password, Q
             }
 
             try (Statement statement = connection.createStatement()) {
-                TimeLimiter limiter = new SimpleTimeLimiter();
+                ExecutorService executor = newSingleThreadExecutor();
```
Comment (dain) [inline presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java]: Maybe make a shared thread pool for this stuff

**Comment by findepi** [comment]
@electrum , did you find a solution for failing tests (e.g accumulo)?
this is were i got stuck some time ago when trying to bump guava version.

**Comment by electrum** [comment]
@findepi I put up the PR to see where the problems are. We might need to shade Accumulo like we did for Cassandra. For the Thrift connector, we are converting it to use Drift. I think I got that working today.

**Comment by electrum** [comment]
@findepi They fixed Guava in the latest Accumulo, thanks to you filing an issue and pushing for it to be fixed.

**Pull request merged**